<?xml version="1.0"?>
<launch>

	<arg name="config_yaml" default="summit_xl"/>
	<arg name="robot_id" default="$(optenv ROBOT_ID robot)"/> 
	<arg name="prefix" default="$(arg robot_id)_"/> 
	<arg name="publish_battery_estimation" default="true"/> 

	<node pkg="robotnik_base_hw_sim" type="robotnik_base_hw_sim_node.py" name="robotnik_base_hw" output="screen">
		<rosparam command="load" file="$(find robotnik_base_hw_sim)/config/$(arg config_yaml).yaml" subst_value="true"/>
	</node>

</launch>
